src/components/progress-report/workflow/progress-report-workflow.service.ts

```diff
@@ -100,6 +126,84 @@ export class ProgressReportWorkflowService {
       this.repo.changeStatus(reportId, transition.to),
     ]);
 
-    // TODO(transition.notify);
+    const notificationInfo = await this.repo.getNotificationInfoByReportId(
+      reportId,
+    );
+    const projectId = notificationInfo[0]?.projectId ?? undefined;
+    const languageDisplayName = notificationInfo[0]?.languageDisplayName;
+    const reportQtrLabel = `Q${fiscalQuarter(
+      notificationInfo[0]?.start,
+    )} FY${fiscalYear(notificationInfo[0]?.start)}`;
+
+    let emailList: string[] = [];
+    notificationInfo.forEach((item) => {
+      item.role?.forEach((role) => {
+        if (transition.notify.TeamMembersWithRoles.includes(role)) {
+          if (item.email) emailList.push(item.email);
+        }
+      });
+    });
+
+    transition.notify.Emails.forEach((element: string) => {
+      if (element.includes('@')) {
+        emailList.push(element);
+      } else if (element === 'MONDAY_EMAIL_DISTRO') {
+        emailList.push(this.configService.mondayEmailDistro);
```

Comment:
This is a magic string. The idea with having `notify` be an object with unique properties was so we didn't have to introspect the values of the properties. Instead this could be a check `transition.notifiy.monday`. Maybe it's just a boolean. 
However since this monday is just an email address, I would instead avoid the word "monday" all together.
Instead the env key could be something like `PROGRESS_REPORT_TRANSITION_NOTIFY_EMAILS`.
It could be dynamic and allow for mapping any transition to any number of emails.

```
Start=address@monday.com,carson_full@tsco.org;Translation Done=carson_full@tsco.org
```
Which could easily be parsed into
```ts
Record<TransitionName, string[]>
```

This allows the specific env to give email addresses for any transition and keeps the codebase here unaware of Monday.com.